runtime/gc_realtime/ProcessorInfo.cpp

```diff
@@ -95,15 +95,15 @@ double
 MM_ProcessorInfo::readFrequency()
 {
 	double freq = 0;
-#if (defined(LINUX) && (defined(J9X86) || defined(J9HAMMER) || defined(LINUXPPC)))
+#if (defined(LINUX) && (defined(J9X86) || defined(J9HAMMER) || defined(LINUXPPC) || defined(J9RISCV64)))
 	char buffer[256];
 	FILE *file = fopen("/proc/cpuinfo", "r");
 	if (file != NULL) {
 		while (!feof(file)) {
 			if (NULL == fgets(buffer, sizeof(buffer), file)) {
 				break;
 			}
-#if (defined(J9X86) || defined(J9HAMMER))
+#if (defined(J9X86) || defined(J9HAMMER) || defined(J9RISCV64))
```

Comment:
I am not sure how it works for RISC-V. However this API is used for Metronome only, so there is no need to implement it now

Reply:
Agreed and removed for now.